Components/Snackbar/snackbar.css

```diff
@@ -0,0 +1,34 @@
+.snackbar{
+    display: flex;
+    align-items: center;
+    /* visibility: hidden; */
+    background-color: #333;
+    color: #fff;
+    text-align: center;
+    border-radius: 6px;
+    padding: 16px;
+    position: fixed;
+    bottom: 10px;
+    left:50%;
+    transform: translate(-50%);
+}
+
+button{
+    color:var(--btn-text-color) ;
+    background-color: var(--primary-color);
+    border:none;
+    padding-left: 2rem;
+    padding-right: 1.5rem;
+    font-weight: bolder;
+    cursor: pointer;
+}
+
+.icon{
+    cursor: pointer;
+}
+
+.icon:hover{
+    border-radius: 50%;
+    background-color: rgb(74, 74, 77);
```

Comment:
try to use css variables for colors

Reply:
yeah in the end i will generalize all the colors from the :root